### DISCUSSION TITLE
Cannot change wallpaper

### DISCUSSION THREAD

**Discussion by ghost** [hyprwm/Hyprland#1190] [Q&A]
I post here since there seems to be no "Discussion" section in the hyprpaper project:
```
❯ hyprctl hyprpaper wallpaper HDMI-A-1,./wallpaper.jpg
Couldn't connect to /tmp/hypr/b8ccf3dc3a03352bccae4cd01dcdd5bc3438c49f_1670395954/.hyprpaper.sock. (3)
```

**Comment by vaxerski** [comment]
means hyprpaper is not running?

**Comment by mshzsh** [comment]
If you could not solve the problem of `hyprpaper`, I suggest you to use `swaybg`.
`swaybg -i /path/to/your/image/image.jpg &`

And if you want GUI, you can install `azote` and use the following command:
`~/.azotebg`

**Reply by vaxerski** [reply]
the problem with swaybg that hyprpaper aims to solve is the wait time before a new wallpaper is applied, which usually results in an empty screen.

**Reply by mshzsh** [reply]
Is true, but the 2 weaknesses that I saw in `hyprpaper` were that, firstly, `hyprpaper` consumes about 7 times more RAM than `swaybg`. and secondly, I could not use fill, fit, center, tile and ... in `hyprpaper`.
Of course, maybe I'm using `hyprpaper` incorrectly, if so please tell me

**Reply by vaxerski** [reply]
> 7 times more RAM than swaybg

odd.

> I could not use fill, fit, center, tile and ... in hyprpaper.

center is by default, you can also use contain
![image](https://user-images.githubusercontent.com/43317083/206866834-e17126ea-a0cf-4135-afba-77c7b544ae1b.png)

**Reply by mshzsh** [reply]
> odd.

On my device it is : hyprpaper 57MB, swaybg 8MB :eyes: 

> center is by default, you can also use contain

Very good, I hope other types like `tile` will be supported soon.

Thanks :+1:
